contrib/inventory/packet_net.py

```diff
@@ -351,7 +351,7 @@ def add_device(self, device, project):
 
         # Inventory: Group by OS
         if self.group_by_operating_system:
-            self.push(self.inventory, device.operating_system.slug, dest)
+            self.push(self.inventory, device.operating_system['slug'], dest)
             if self.nested_groups:
                 self.push_group(self.inventory, 'operating_systems', device.operating_system.slug)
 
```

Comment:
Would `device.operating_system.slug` on line 356 also need to be updated to `device.operating_system['slug']`?

Reply:
Yes. You are correct. I'll fix that now.